src/Event/BookingEvent.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Event;
+
+use App\Entity\Ticket;
+
+class BookingEvent
+{
+
+    public function __construct(Ticket $ticket)
+    {
+    }
+}
```

Comment:
huh? What event is this? What do you get by this event?

Reply:
I think I should drink some coffee